test/source_code_test.rb

```diff
@@ -100,6 +100,7 @@ def self.token_is_keyword?(str, lex, index, token)
       DOC
       SourceCode.
           new('**/*.{rb,js,scss,haml}',
+              'foreman/**',
```

Comment:
Does this mean we're looking for malformed strings in `foreman`? Don't their tests do that?

Reply:
Anything after the first line is 'ignores': https://github.com/Katello/katello/blob/master/test/source_code_test.rb#L22

with the new ci testing, the foreman directory was checked out within the katello dir, so we need to ignore it.